Simplified/Reader2/Bookmarks/NYPLBookmarkFactory.swift

```diff
@@ -80,13 +81,25 @@ class NYPLBookmarkFactory {
       device: drmDeviceID)
   }
 
+  class func make(fromServerBookmark annotation: [String: Any],
+                  bookID: String) -> NYPLReadiumBookmark? {
+    return make(fromServerAnnotation: annotation,
+                annotationType: .bookmark,
+                bookID: bookID)
+  }
+
   class func make(fromServerAnnotation annotation: [String: Any],
+                  annotationType: NYPLBookmarkSpec.Motivation,
```

Comment:
this is now the main parsing function from server data for both reading position and explicit bookmarks